scripts/install-snapshot.sh

```diff
@@ -77,6 +77,29 @@ function delete_snapshot_crd() {
     kubectl delete -f "${VOLUME_SNAPSHOT}" --ignore-not-found
 }
 
+# parse the kubernetes version
+# v1.17.2 -> kube_version 1 -> 1  (Major)
+# v1.17.2 -> kube_version 2 -> 17 (Minor)
+function kube_version() {
+    echo "${KUBE_VERSION}" | sed 's/^v//' | cut -d'.' -f"${1}"
+}
+
+KUBE_VERSION=$(kubectl version --short 2>/dev/null | grep "^Server Version" | cut -d' ' -f3)
+if [[ -z ${KUBE_VERSION} ]]; then
+    echo "could not get kube server version"
+    echo "hint: check if you have specified the right host or port"
+    exit 1
+fi
+
+KUBE_MAJOR=$(kube_version 1)
+KUBE_MINOR=$(kube_version 2)
+
+# skip snapshot operation if kube version is less than 1.17.0
+if [[ "${KUBE_MAJOR}" -lt 1 ]] || [[ "${KUBE_MINOR}" -lt 17 ]]; then
```

Comment:
I think this fails when Kubernetes 2.0 gets released?

Reply:
Ah! good catch, Will fix it right away.